app/reducers/antibioticConsumptionStatsList.js

```diff
@@ -17,11 +20,22 @@ const initialState = {
 };
 
 export default function(state: State = initialState, action: Action) {
```

Comment:
Please add a `CLEAN_ANTIBIOTIC_CONSUMPTION_STATS_LIST` action, trigger it from the page component `willUnmount` and handle it here by setting the state to its initial value. This is important to simplify reasoning about the reducer. Especially considering that `ADD_CREATED_ANTIBIOTIC_CONSUMPTION_STAT_TO_LIST` does some pretty ad-hoc transformations.